Makefile

```diff
@@ -4,13 +4,39 @@ DOCKER_BUILDKIT=1
 SHELL=/bin/bash
 CWD=$(shell pwd)
 PULL ?=True
+DEV ?=n
```

Comment:
Please keep this logic contained in the physics directory. We'll need to be able to develop and test it as a self-contained directory in the long term (not in this PR), and this is a lot of its low level logic to be putting at the top level for now.